pom.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>3.4.0</version>
+        <relativePath/>
+    </parent>
+    <groupId>eu.ai4work</groupId>
+    <artifactId>sliding-work-sharing</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>Sliding Work Sharing</name>
+    <description>Sliding Work Sharing management component of the AI4Work project</description>
+
+    <packaging>jar</packaging>
+
+    <inceptionYear>2024</inceptionYear>
+
+    <url>https://www.atb-bremen.de</url>
+
+    <organization>
+        <name>ATB – Institut für angewandte Systemtechnik Bremen GmbH</name>
+        <url>https://www.atb-bremen.de</url>
+    </organization>
+
+    <properties>
+        <maven.compiler.source>23</maven.compiler.source>
+        <maven.compiler.target>23</maven.compiler.target>
+        <java.version>23</java.version>
+        <drools.version>10.0.0</drools.version>
+        <kie.version>10.0.0</kie.version>
+        <lombok.version>1.18.36</lombok.version>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-actuator</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-devtools</artifactId>
+            <scope>runtime</scope>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <!-- Drools decision tables, for to define the rules -->
+        <dependency>
+            <groupId>org.drools</groupId>
+            <artifactId>drools-decisiontables</artifactId>
+            <version>${drools.version}</version>
+        </dependency>
+        <!-- Kubernetes Integrated Environment  -->
```

Comment:
Are we actually using this?

Reply:
yes, to work with Drools `kie-ci` is needed, specifically for classes like `KieServices` etc. It is important to managing rule-releted configurations.